### ISSUE TITLE
Examples not working

### ISSUE THREAD

**Issue by cl3i550n** [react-native-maps/react-native-maps#4189]
Examples not working after 

```
1. cd examples/
2. npm install
3. npx react-native run-android
```

```
$ npx react-native run-android
info Running jetifier to migrate libraries to AndroidX. You can disable it using "--no-jetifier" flag.
Jetifier found 877 file(s) to forward-jetify. Using 8 workers...
info Starting JS server...
info Installing the app...
Downloading https://services.gradle.org/distributions/gradle-6.9-all.zip
..............10%..............20%...............30%..............40%..............50%...............60%..............70%...............80%..............90%..............100%

Welcome to Gradle 6.9!

Here are the highlights of this release:
 - This is a small backport release.
 - Java 16 can be used to compile when used with Java toolchains
 - Dynamic versions can be used within plugin declarations
 - Native support for Apple Silicon processors

For more details see https://docs.gradle.org/6.9/release-notes.html

Starting a Gradle Daemon (subsequent builds will be faster)

FAILURE: Build failed with an exception.

* What went wrong:
Could not open settings generic class cache for settings file 'C:\Users\cleis\Documents\MyGitHub\react-native-maps\example\android\settings.gradle' (C:\Users\cleis\.gradle\caches\6.9\scripts\7uc7lki81pf5228csk1w5j12y).
> BUG! exception in phase 'semantic analysis' in source unit '_BuildScript_' Unsupported class file major version 62

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 20s

error Failed to install the app. Make sure you have the Android development environment set up: https://reactnative.dev/docs/environment-setup.
Error: Command failed: gradlew.bat app:installDebug -PreactNativeDevServerPort=8081

FAILURE: Build failed with an exception.

* What went wrong:
Could not open settings generic class cache for settings file 'C:\Users\cleis\Documents\MyGitHub\react-native-maps\example\android\settings.gradle' (C:\Users\cleis\.gradle\caches\6.9\scripts\7uc7lki81pf5228csk1w5j12y).
> BUG! exception in phase 'semantic analysis' in source unit '_BuildScript_' Unsupported class file major version 62

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 20s

    at makeError (C:\Users\cleis\Documents\MyGitHub\react-native-maps\example\node_modules\execa\index.js:174:9)
    at C:\Users\cleis\Documents\MyGitHub\react-native-maps\example\node_modules\execa\index.js:278:16
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async runOnAllDevices (C:\Users\cleis\Documents\MyGitHub\react-native-maps\example\node_modules\@react-native-community\cli-platform-android\build\commands\runAndroid\runOnAllDevices.js:106:5)
    at async Command.handleAction (C:\Users\cleis\Documents\MyGitHub\react-native-maps\example\node_modules\react-native\node_modules\@react-native-community\cli\build\index.js:192:9)
info Run CLI with --verbose flag for more details.

```

**Comment by salah-ghanim** [comment]
@cl3i550n there is a pull request to fix this #4170 ... waiting to be merged

**Comment by NamanAg0502** [comment]
Same error occurring with me...
